CHANGES.rst

```diff
@@ -14,7 +14,14 @@ New features:
 
 Bug fixes:
 
-- *add item here*
+- #1794: use permissive vocabulary type for numeric-indexed weekday and
+  month vocabularies to avoid breaking GenericSetup import.
+  [seanupton]
+
+- #1268: use permissive vocabulary type for ReallyUserFriendlyTypes,
```

Comment:
i guess it would be better to reference the ticket number like this since it's in another package.
`plone/Products.CMFPlone#1268`